bridge/contracts/Federation/FederationV3.sol

```diff
@@ -224,8 +224,10 @@ contract FederationV3 is Initializable, UpgradableOwnable, IFederationV3 {
         value,
         blockHash,
         transactionHash,
-        logIndex
-      );
+        logIndex,
+        block.chainid,
+        block.chainid
```

Comment:
kkk oh ya, we intend to remove the v3, I was hoping to be able to merge this first, but if you prefer I could create another PR on top of this one to do it

Like, the v4 would be the v3, because the v3 was never released into main net

Reply:
creathe the Pr on top of this while we merge the previous ones

Reply:
Will do